### PR TITLE
Updated CONTRIBUTIONS.md  to make it more friendly with specific URLs and commands to run

### PR DESCRIPTION
**Pull request by Sreedhar1895**
I have modified the CONTRIBUTIONS.md by adding necessary URLs and commands to run to make the document more friendly.
The URLs provided help the user to install the dependencies required.
I have also provided URLs that will provide the user links to pages to know more about the dependencies.
I have also added the necessary commands to run.

Cheers!

### CODE REVIEW

**Comment by creativepsyco** [comment]
Hi @Sreedhar1895,

Please follow markdown format for links. Also you have 2 files in the commit which I am not sure if it was the intention. And you have multiple commits. Squash them into 1.

**Comment by Sreedhar1895** [comment]
Alright :)

File: CONTRIBUTING.md~ [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Contributing to our project
```
Comment (creativepsyco) [inline CONTRIBUTING.md~]: remove this swap file.

**Comment by creativepsyco** [comment]
Also please write detailed commands which someone would execute to get stuff running on their computer not just a bunch of links which can be found through google ;)

**Comment by Sreedhar1895** [comment]
Alright I'll make the necessary changes and send a pull request again :)

I have vagrant installed and can create the local environment so how do I
go about running the programs on my local machine to get involved in
finding tasks and contributing further? :)

I'm new to open source so help is appreciated :)

**Comment by creativepsyco** [comment]
@Sreedhar1895 you look up the list of issues to see what is outstanding and what you can help with.

**Comment by Sreedhar1895** [comment]
Hey I have updated and made the necessary changes.

Should I send a new pull request?

**Comment by creativepsyco** [comment]
There was another PR for the same which has been merged, please check the file again. I am closing this PR.

**Comment by Sreedhar1895** [comment]
Alright! :+1:

**Pull request closed without merging**